src/dynarec/rv64/dynarec_rv64_0f.c

```diff
@@ -200,7 +200,43 @@ uintptr_t dynarec64_0F(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
             SD(x3, wback, fixedaddress+0);
             SD(x4, wback, fixedaddress+8);
             break;
-
+        case 0x2E:
+            // no special check...
+        case 0x2F:
+            if(opcode==0x2F) {INST_NAME("COMISS Gx, Ex");} else {INST_NAME("UCOMISS Gx, Ex");}
+            SETFLAGS(X_ALL, SF_SET);
+            nextop = F8;
+            GETGX(x3);
+            GETEXSS(v0, 0);
+            CLEAR_FLAGS();
+            // if isnan(gd) || isnan(v0)
+            IFX(X_ZF | X_PF | X_CF) {
+                FLW(gd, x3, 0);
+                FEQS(x3, gd, gd);
+                FEQS(x2, v0, v0);
+                AND(x2, x2, x3);
+                XORI(x2, x2, 1);
+                BEQ_MARK(x2, xZR);
+                ORI(xFlags, xFlags, (1<<F_ZF) | (1<<F_PF) | (1<<F_CF));
+                B_NEXT_nocond;
+            }
+            MARK;
+            // else if isless(gd, v0)
+            IFX(X_CF) {
+                FLTS(x2, gd, v0);
+                BEQ_MARK2(x2, xZR);
+                ORI(xFlags, xFlags, 1<<F_CF);
+                B_NEXT_nocond;
+            }
+            MARK2;
+            // else if gd == v0
+            IFX(X_ZF) {
+                FEQS(x2, gd, v0);
+                BEQ_MARK3(x2, xZR);
```

Comment:
a branch to NEXT could be used here also...

Reply:
Yeah, will do it in next PR.